lib/jxl/enc_frame.cc

```diff
@@ -459,6 +460,199 @@ void SimplifyInvisible(Image3F* image, const ImageF& alpha, bool lossless) {
   }
 }
 
+constexpr size_t kSeparationRes = 32;
```

Comment:
I'm not sure enc_frame is the best place for this code, I'd maybe make a new file.

Reply:
Makes sense. Could also see it as lossy heuristics or part of patches, I guess. But separate file is maybe best.